modules/creating-a-machine-pool-ocm.adoc

```diff
@@ -17,15 +17,15 @@ ifndef::openshift-rosa,openshift-rosa-hcp[]
 A machine pool is created when you install an {product-title} cluster. After installation, you can create additional machine pools for your cluster by using {cluster-manager}.
 endif::openshift-rosa,openshift-rosa-hcp[]
 ifdef::openshift-rosa,openshift-rosa-hcp[]
-You can create additional machine pools for your {product-title} (ROSA) cluster by using {cluster-manager}.
+You can create additional machine pools for your {product-title} cluster by using {cluster-manager}.
 endif::openshift-rosa,openshift-rosa-hcp[]
 
 ifndef::openshift-rosa,openshift-rosa-hcp[]
 [IMPORTANT]
 ====
 The compute (also known as worker) node instance types, autoscaling options, and node counts that are available depend on your
 ifdef::openshift-rosa,openshift-rosa-hcp[]
```

Comment:
Suggest removing the conditionals here for both ROSAs as now it' s just {product-title} for all OpenShift flavors.